tests/series/test_image.py

```diff
@@ -0,0 +1,55 @@
+from __future__ import annotations
+
+import copy
+
+import numpy as np
+
+from daft.datatype import DaftExtension, DataType
+from daft.series import Series
+
+
+def test_image_arrow_round_trip():
+    data = [np.arange(4).reshape((1, 2, 2)), np.arange(4, 13).reshape((1, 3, 3)), None]
+    s = Series.from_pylist(data, pyobj="force")
+
+    target_dtype = DataType.image("F")
```

Comment:
I would drop the F type since it typically is only 1 channel and rely on the enum like `RGB32F`